template.go

```diff
@@ -129,6 +132,13 @@ func readYAMLRequest(base string, contents []byte, templateArgs map[string]strin
 		return err
 	}
 
+	if t.File != "" {
+		body, err = ioutil.ReadFile(t.File)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
We should prevent users of using `file` and `request` together. Right now, File have priority over request, only one should be available